terraform/security.tf

```diff
@@ -55,15 +55,36 @@ resource "aws_security_group" "ecs_tasks" {
 }
 
 resource "aws_security_group" "db" {
-  name        = "beacons-rds-security-group"
+  name        = "${var.env}-beacons-rds-security-group"
   description = "Allows inbound access from the ECS tasks only"
   vpc_id      = aws_vpc.main.id
 
   ingress {
     protocol        = "tcp"
     from_port       = 5432
     to_port         = 5432
-    security_groups = aws_security_group.ecs_tasks.*.id
+    security_groups = aws_security_group.ecs_tasks[*].id
+  }
+
+  egress {
+    protocol    = "-1"
+    from_port   = 0
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+# Allows traffic from the ECS cluster to VPC endpoints for ECR images
```

Comment:
Following [this guide](https://aws.amazon.com/blogs/compute/setting-up-aws-privatelink-for-amazon-ecs-and-amazon-ecr/) - is at the end of the `Create an AWS PrivateLink interface endpoint for ECR` step